test/builder/QuarkBuilderTransfer.t.sol

```diff
@@ -48,40 +48,53 @@ contract QuarkBuilderTransferTest is Test, QuarkBuilderTest {
 
     function testInsufficientFunds() public {
         QuarkBuilder builder = new QuarkBuilder();
-        vm.expectRevert(abi.encodeWithSelector(QuarkBuilder.InsufficientFunds.selector, 10e6, 0e6));
+        vm.expectRevert(abi.encodeWithSelector(QuarkBuilder.FundsUnavailable.selector, "USDC", 10e6, 0e6));
         builder.transfer(
-            transferUsdc_(1, 10e6, address(0xfe11a), BLOCK_TIMESTAMP), // transfer 10USDC on chain 1 to 0xfe11a
+            transferUsdc_(1, 10e6, address(0xfe11a), BLOCK_TIMESTAMP), // transfer 10 USDC on chain 1 to 0xfe11a
             chainAccountsList_(0e6), // but we are holding 0 USDC in total across 1, 8453
             paymentUsd_()
         );
     }
 
+    // TODO: This no longer tests for the MaxCostTooHigh, since it may be unreachable now. Verify that this is the case.
     function testMaxCostTooHigh() public {
         QuarkBuilder builder = new QuarkBuilder();
-        vm.expectRevert(QuarkBuilder.MaxCostTooHigh.selector);
+        // Max cost is too high, so total available funds is 0
+        vm.expectRevert(abi.encodeWithSelector(QuarkBuilder.FundsUnavailable.selector, "USDC", 1e6, 0e6));
         builder.transfer(
-            transferUsdc_(1, 1e6, address(0xfe11a), BLOCK_TIMESTAMP), // transfer 1USDC on chain 1 to 0xfe11a
+            transferUsdc_(1, 1e6, address(0xfe11a), BLOCK_TIMESTAMP), // transfer 1 USDC on chain 1 to 0xfe11a
             chainAccountsList_(2e6), // holding 2 USDC in total across 1, 8453
-            paymentUsdc_(maxCosts_(1, 1_000e6)) // but costs 1,000USDC
+            paymentUsdc_(maxCosts_(1, 1_000e6)) // but costs 1,000 USDC
         );
     }
 
-    function testFundsUnavailable() public {
+    function testFundsOnUnbridgeableChains() public {
         QuarkBuilder builder = new QuarkBuilder();
-        // FundsUnavailable(2e6, 0e6, 2e6): Requested 2e6, Available 0e6, Still missing 2e6
+        // FundsUnavailable(2e6, 0e6, 2e6): Requested 2e6, Available 0e6
```

Comment:
Nit: I think that first argument in your comment should be the assetSymbol:

```
error FundsUnavailable(string assetSymbol, uint256 requiredAmount, uint256 actualAmount);
```

Reply:
Done. Thanks